lib/model.js

```diff
@@ -4100,9 +4100,12 @@ Model.validate = async function validate(obj, pathsToValidate, context) {
   }
   let paths = Object.keys(schema.paths);
 
-  if (pathsToValidate != null) {
-    const _pathsToValidate = typeof pathsToValidate === 'string' ? new Set(pathsToValidate.split(' ')) : new Set(pathsToValidate);
+  if (pathsOrOptions != null) {
+    const _pathsToValidate = typeof pathsOrOptions === 'string' ? new Set(pathsOrOptions.split(' ')) : Array.isArray(pathsOrOptions) ? new Set(pathsOrOptions) : new Set(paths);
```

Comment:
In the case where `pathsOrOptions = { pathsToSkip: ['name'] }`, then `paths` will equal `['pathsToSkip']`, which is incorrect.